executor/executable/controllabletask.go

```diff
@@ -417,13 +418,13 @@ func (t *ControllableTask) Launch() error {
 			}
 			for {
 				if t.rpc == nil {
-					log.WithField("partition", t.knownEnvironmentId.String()).
+					log.WithField("partition", t.knownEnvironmentId.String()).WithField("taskId", deo.TaskId.String()).
```

Comment:
Can you please wrap each `WithField` on its own line? I know it's a silly nitpick but it makes these log statements much more readable.
Otherwise we're good to go :rocket:

Reply:
no worries, fair point!